### ISSUE TITLE
Status bar area changed after banner show

### ISSUE THREAD

**Issue by yigit-serin** [admob-plus/admob-plus#310]
**Describe the bug**
Status bar area changed after banner show in IOS.

**To Reproduce**
1. Run app which is contains tabs in IOS simulator, IOS version: 14.4, Simulator: Iphone XR
2. Open a new page without tab.
3. Create and show banner ad with default params.
4. new admob.BannerAd({
        adUnitId: 123,
      });
4. Click back button.
5. banner.hide();
6. Status bar area changed, status bar color changed.

**Expected behavior**
There should be no change.

**Screenshots**
<img width="1053" alt="Screen Shot 2021-04-16 at 01 15 04" src="https://user-images.githubusercontent.com/20225497/114946085-32ff5300-9e53-11eb-8f38-a81934e22846.png">
<img width="1047" alt="Screen Shot 2021-04-16 at 01 16 00" src="https://user-images.githubusercontent.com/20225497/114946107-3a266100-9e53-11eb-9ab1-33e191db5d32.png">
**Environment**
```
  AdMob Plus Environment Info:
    Binaries:
      Node: 12.18.2 - /usr/local/bin/node
      Yarn: 1.22.10 - /usr/local/bin/yarn
      npm: 6.14.11 - /usr/local/bin/npm
    Managers:
      CocoaPods: 1.10.1 - /usr/local/bin/pod
    Utilities:
      Clang: 12.0.0 - /usr/bin/clang
    SDKs:
      iOS SDK:
        Platforms: iOS 14.4, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
      Android SDK:
        API Levels: 26, 27, 28, 29, 30
        Build Tools: 28.0.3, 29.0.2, 30.0.1
        System Images: android-28 | Google APIs Intel x86 Atom, android-29 | Google APIs Intel x86 Atom, android-29 | Google Play Intel x86 Atom
    npmPackages:
      admob-plus-cordova: ^1.5.4 => 1.5.4 
      cordova-android: ^9.0.0 => 9.1.0 
      cordova-ios: ^6.0.1 => 6.2.0 
      typescript: ~2.6.2 => 2.6.2 
    npmGlobalPackages:
      cordova: 10.0.0
    Cordova:
      Version: 10.0.0
    Ionic:
      Version: 6.12.3
```

**Additional context**
Admob banner trigger something and changed status bar area or padding or something.
Any suggestion to fix this?

**Comment by ratson** [comment]
@yigit-serin There is really no easy fix for this issue as it come down to who is managing the webview size.
Use `offset` could bypass the problem by not resizing the webview, then within the webview reserve space for displaying the banner.
I am going to add a new API to control the top/bottom margin for those need it.

**Comment by yigit-serin** [comment]
@ratson I tried with **offset** but status bar changed again, no solution. I think adding the banner view causes this situation independent of location. (over content or before content)
I tried new banner config parameters but I can't see any difference.
` this.banner = new admob.BannerAd({
        adUnitId: 123
      });
      await this.banner.show();
      this.banner.config({marginBottom: 100});`

if I use like this;
` this.banner = new admob.BannerAd({
        adUnitId: 123
      });
      this.banner.config({marginBottom: 100});
      await this.banner.show();`
no ad displayed.

**Comment by ratson** [comment]
> I tried with offset but status bar changed again, no solution. I think adding the banner view causes this situation independent of location. (over content or before content)

Should be working with `admob-plus-cordova@1.7.0`

> if I use like this;
this.banner = new admob.BannerAd({ adUnitId: 123 }); this.banner.config({marginBottom: 100}); await this.banner.show();
no ad displayed.

It is a static method, see https://admob-plus.github.io/docs/cordova/ads/banner#global-settings

**Comment by bardgr** [comment]
@ratson tried with 1.7.0 did not work
It is the same with the issue https://github.com/admob-plus/admob-plus/issues/297
that was solved with 1.5.1 version
